### DISCUSSION TITLE
Can't get watch command to work

### DISCUSSION THREAD

**Discussion by benthien** [foundation/proton#20] [General]
Has anyone been able to get the watch command to work in Proton? If so I would like to see an example. The following is a description of my failed attempt

I performed the following steps in Windows 11 WSL, and in a standalone version of Linux-mint 20. In both I get the same error message.

1. created a directory temp
2. ran the command `proton init --template=sites`
3. ran the command `yarn`
4. ran the command `yarn add browser-sync chokidar -D`
5. ran the command `proton watch`

Here is a screenshot after step 2 (ubuntu in wsl)

![step2](https://user-images.githubusercontent.com/12056197/156655329-ef544f00-b235-4c73-951d-12db7d8ca534.png)

Here is a screen shot after steps 3 and 4

![steps3and4](https://user-images.githubusercontent.com/12056197/156655490-48c91ad7-60e2-447a-97d1-1a04706abd06.png)

Here is a screen shot showing the error after step 5 

![error_in_step5](https://user-images.githubusercontent.com/12056197/156655672-cdfcf086-2787-4459-b239-58de77244123.png)

However, `proton build` worked properly.

**Comment by joeworkman** [comment]
Were you able to figure this out?

**Reply by benthien** [reply]
No, not yet. Has anyone else had this problem?

**Reply by joeworkman** [reply]
I was able to replicate it today. I don't have a solution yet though.
